syntax = "proto2";

package login;

message Person 
{
  required string name = 1;
  required string email = 2;
  required string location = 3;
  required string password = 4;
  required int32 userid = 5;
}

message PeopleBook {
  repeated Person people = 1;
}